test/move_effect_double_shock.c

```diff
@@ -0,0 +1,57 @@
+#include "global.h"
+#include "test_battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_DOUBLE_SHOCK].effect == EFFECT_DOUBLE_SHOCK);
+}
```

Comment:
Needs the assumption that Pikachu is an Electric type and that Wobbuffet is not.